cmd/manager/main.go

```diff
@@ -143,6 +144,8 @@ func newManager(ctx context.Context, restCfg *rest.Config, scopedNamespace strin
 
 	controllersCfg.UserProjectOverride = userProjectOverride
 	controllersCfg.BillingProject = billingProject
+	// TODO(b/320784855): StateIntoSpecDefaultValue and StateIntoSpecUserOverride values should come from the flags.
```

Comment:
At some point we are going to read the CCC directly in the reconciler rather than relaunching.  So we don't need to plumb this through the operator etc.